README.md

```diff
@@ -103,7 +103,7 @@ or a file path like [`tasks_examples/recommended_set.txt`](./tasks_examples/reco
 ```shell
 accelerate launch --multi_gpu --num_processes=8 run_evals_accelerate.py \
     --model_args "pretrained=gpt2" \
-    --tasks "lighteval|truthfulqa:mc|0|0" \
+    --tasks "leaderboard|truthfulqa:mc|0|0" \
```

Comment:
Maybe it's good to showcase the `lighteval` suite here and then point people to the `leaderboard` one later in the README (e.g. where we point them to the leaderboard tasks.txt)?

Otherwise I think users will get confused about which suite to use as a default and promoting `lighteval` makes most sense IMO

Reply:
SGTM